kuma/wiki/jinja2/wiki/includes/document_macros.html

```diff
@@ -77,6 +77,12 @@
         <dd class="rev-to"> {{ ', '.join(revision_to.tags.split(',')) }}</dd>
         {% endif %}
 
+        {% if revision_from.comment or revision_to.comment %}
+        <dt>{{ _('Comment:') }}</dt>
```

Comment:
I followed the indent used at the previous block of code.
If @stephaniehobson or @jwhitlock wants this indented in an other way, it would implie to reindent the rest of the file, I suppose.